### PR TITLE
[WIP] Conference page update

### PR DESCRIPTION
**Pull request by mkalininait**

### CODE REVIEW

File: students/conferences.md [hunk 1]
```diff
@@ -9,148 +9,204 @@ current: students
...
+We contact conference organizers and ask for support. All the tickets we get, we distribute between teams based on the following criteria:  
 
-Usually, students pick conferences that are close by. We also favor this for financial reasons (LINK TO TRAVEL&ACCOMMODATION). From the list of ticket applications, we distribute the tickets with these criteria:
-* order of wish list
-* suitable conference (location/financial aspects, topic of conference)
-* fair distribution
+* teams' preferences
 
-Having said that, if you are able to attend a conference at a far-away place, feel free to apply for one.
+* suitabitily (location/financial aspects, topic of conference)
```
Comment (alicetragedy) [inline students/conferences.md]: small typo: suitability

File: students/conferences.md [hunk 2]
```diff
@@ -9,148 +9,204 @@ current: students
...
 
-If you won tickets for a conference, we will connect you with the conference organizers. You can then directly talk about the details of how to actually get your personal tickets. Depending on how it’s done, you might need to sign-up somewhere or simply provide your names so the organizers can put you on the list.
+Usually we recommend students pick local conferences -- for financial reasons. However, if you are able to attend a conference at a far-away place, feel free to apply for one.
+
+We announce conferences and teams associated with them in two goes. After your conference is officially announced, we connect you with the conference organizers. You can then directly talk about the details of how to actually get your personal tickets. Depending on how it’s done, you might need to sign-up somewhere or simply provide your names so the organizers can put you on the list.
```
Comment (alicetragedy) [inline students/conferences.md]: replace "goes" with "rounds" (I think it's easier to understand).

File: students/conferences.md [hunk 3]
```diff
@@ -9,148 +9,204 @@ current: students
...
+We also have in mind that it will probably be the first public talk for you. Scary thought, right? We know. Stage fright is something we all experience, even if we already have held a lot of talks. We will help you in dealing with it! Here’s a start: <a href="http://railsgirlssummerofcode.org/blog/2014-07-29-talk-tips">How to ace your lightning talk</a>.
 
 <h2 id="travel">What About Travel and Accommodation?</h2>
-Unfortunately, we are not able to provide additional financial support for travel & accommodation. If we see an opportunity, we will ask the conference organizers about it. But please consider that providing travel grants is a big effort for everyone involved and not usually happening.
+Unfortunately, we are not able to provide additional financial support for travel & accommodation. If we see an opportunity, we will ask conference organizers about it, but please consider that providing travel grants is a big effort for everyone involved and happens not that often.
```
Comment (alicetragedy) [inline students/conferences.md]: "happens not that often" => "doesn't happen that often"

File: students/conferences.md [hunk 4]
```diff
@@ -9,148 +9,204 @@ current: students
...
 
 There are a few things to consider though:
 
 * Your conference should be beginner friendly. That means, talks (or workshops) need to be suitable for people who haven’t done this for years.
 
-* You take care of a welcoming, respectful and harassment-free atmosphere. We expect you to either have a Code of Conduct in place or otherwise make an effort in dealing with - let us be plain - jerk behavior. Bonus points go to conferences who have assigned contact persons for potential incidents.
-
-At the conference, we want our students to hold a lightning talk about their project. If you have slots for this, we’d be very happy if a team could use one of them. If not, no worries. Alternatively, they can write about their conference experience on our blog.
+* You take care of a welcoming, respectful and harassment-free atmosphere. We expect you to either have a Code of Conduct in place or otherwise make an effort in dealing with -- let us be plain -- jerk behavior. Bonus points go to conferences who have assigned contact persons for potential incidents.
```
Comment (alicetragedy) [inline students/conferences.md]: We now don't accept conferences without a code of conduct, so it should be made clearer here. I suggest replacing this sentence with:

"We expect you to have a Code of Conduct in place and to make an effort in dealing with -- let us be plain -- jerk behavior."

**Comment by alicetragedy** [comment]
Hey @mkalininait, the improvements on the page look great! I've made a couple of comments for corrections (mainly to make some of the points clearer and for a couple of typos). 
I also love the new look for the conferences section (with the logos <3), but we try to put as little inline styling on the pages as possible, because it starts to become very unmaintainable if sections are styled in their own pages. All the css you added should be, if possible, moved to one of the external stylesheets. Can you do that? We can talk about it more in detail if you need help or to pair with someone 👍

**Comment by mkalininait** [comment]
@alicetragedy Thank you for reviewing the changes. All your comments make sense.

**Comment by mkalininait** [comment]
Hey @alicetragedy I've moved styles to layout.scss. 
I decided to keep conferences in multiple tables for the moment (not one table and not divs). I'm not sure if this is the right thing to do, but it seems pretty flexible and I don't have a good argument why I should change it.

**Comment by alicetragedy** [comment]
@mkalininait it's good like this! awesome work. it's ready to be merged from my side 👍

**Pull request merged**